assests/javascript/index.js

```diff
@@ -0,0 +1,73 @@
+const baseUrl = "http://127.0.0.1:3030";
+
+const newTask = document.getElementById("task-input");
+const addTaskIcon = document.getElementById("add-task-icon");
+const taskBox = document.getElementById("task-box");
+const newTaskErrorMassage = document.getElementById("title-error");
+const networkErrorMassage = document.getElementById("network-error");
+
+newTask.addEventListener(
+  "keydown",
+  (event) => {
+    if (event.key === "Enter") {
+      handleNewTask(newTask.value);
+      newTask.value = "";
+    }
+  },
+  false
```

Comment:
Could you please explain this a bit here?

Reply:
Its an optional part of `addeventlistener`.
and default is `false` so it doesn't matter to write it or not.